participants/1pan.md

```diff
@@ -0,0 +1,69 @@
+# <Project Name>
+
+Before submitting a PR containing your team's project information:
+
+- Make sure you rename this file to contain your project's name
+- Respond to add your response in each section below
+
+## (0) Project Name and Team Slack Handles
+
+Name:1pan
+Slack:longkengkeng
+
+MemberId:U01BZR3FNPJ
+
+## (1) Brief Project Description
+
+*We wanted to create a disk application on the Filecoin network so that our user data would not be lost. For the foreseeable future, we hope that our application data in the Filecoin gateway will be easily accessible to our users.*
+
+## (2) Link to Project UI
+
+*demo: http://ipfs.forke.cn/*
+
+You can see UI in github:
+
+https://github.com/zlongxiao/1pan.git
+
+## (3) What does your application/UI do?
+
+*This is a traditional disk project, and we really want to see it become a business application on the Filecoin network.The project has login, registration, upload, share, download, reward and other functions.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*Various*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
+*We will not collect any data, and we will slice the data and store it in the Filecoin network. In the IPFS network, there is no complete data in our application, and the data uploaded to the IPFS network will be reviewed by our staff.*
+
+## (6) How much data are you planning to store to the Filecoin network during the Slingshot competition?
+
+*We now have almost 500G of data, and we plan to upload some of the cold data to the IPFS network. During the use, if the access speed can reach our expectation, we will upload all the data uploaded by users to the IPFS network.*
+
+## (7) How are you structuring the data?
+
+*We cut the data into pieces and store it on the Filecoin network.*
```

Comment:
This question is asking more about what the data structure looks like. Is it just binary data? Text files with some metadata? Do you have a data schema for what you're storing? Can you please provide more detail here?

Reply:
Yes ,with the file's unique heads binary data.Should I put it in my md file?